test/zipObjWith.js

```diff
@@ -0,0 +1,39 @@
+import { assert } from 'chai';
+import * as R from 'ramda';
+
+import * as RA from '../src';
+
+describe('unzipObjWith', function() {
+  it('unzips an object into key/value arrays applying a value/key transform', function() {
```

Comment:
Use tests conventions already utilized in `unzipObjWith`. `should/context/specify` etc...